api/resmap/reswrangler.go

```diff
@@ -530,7 +530,7 @@ func (m *resWrangler) Select(s types.Selector) ([]*resource.Resource, error) {
 		// matches the namespace when namespace is not empty in the selector
 		// It first tries to match with the original namespace
 		// then matches with the current namespace
-		if r.GetNamespace() != "" {
```

Comment:
Thanks for finding this.

Can you add one more test that would have failed before this change and passes with this change?